SharedPackages/BrowserServicesKit/Package.swift

```diff
@@ -462,6 +463,12 @@ let package = Package(
                 .define("DEBUG", .when(configuration: .debug))
             ]
         ),
+        .target(
+            name: "PixelExperimentKitTestUtils",
+            dependencies: [
+                "PixelExperimentKit",
+            ]
+        ),
```

Comment:
Please refer to this comment for more information on this change: https://github.com/duckduckgo/apple-browsers/pull/562/files#r2055883698

I needed to make these mocks available to the main app.